distributed/security.py

```diff
@@ -153,18 +153,57 @@ def _set_field(self, kwargs, field, config_name):
             out = dask.config.get(config_name)
         setattr(self, field, out)
 
-    def __repr__(self):
+    def attr_to_dict(self):
```

Comment:
This probably wants to be private.

```suggestion
    def _attr_to_dict(self):
```

Reply:
Oh right ✔️